chainer/training/extensions/parameter_statistics.py

```diff
@@ -0,0 +1,233 @@
+import numpy
+
+import cupy
```

Comment:
Don't import cupy directly because chainer needs to support an environment without GPU. Instead, use `cuda.cupy`.